Jenkinsfile-test-binaries

```diff
@@ -45,7 +45,7 @@ def checkoutRepository(os_type) {
 pipeline {
     agent none
     parameters {
-        string(defaultValue: 'master', name: 'git_branch',
+        string(defaultValue: 'PR-829', name: 'git_branch',
```

Comment:
I am guessing this isnt needed?

Reply:
Ah, must have changed it to debug. Gonna rever that now.